src/languages/handlebars.js

```diff
@@ -5,32 +5,58 @@ Author: Robin Ward <robin.ward@gmail.com>
 Description: Matcher for Handlebars as well as EmberJS additions.
 Category: template
 */
-
 function(hljs) {
   var BUILT_INS = {'builtin-name': 'each in with if else unless bindattr action collection debugger log outlet template unbound view yield'};
+  function MUSTACHE_CONTENTS() {
+    return  {
+      className: 'name',
+      begin: /[a-zA-Z\.-]+/,
+      keywords: BUILT_INS,
+      starts: {
+        endsWithParent: true, relevance: 0,
+        contains: [
+          hljs.QUOTE_STRING_MODE
```

Comment:
Is this for literal params?  Should we have a test for that?

Reply:
Yes we should. And it is not working correctly (https://jsfiddle.net/abyejr70/), neither in the latest version nor in my PR. 

But I would like to create another PR for that.